controllers/funding/materials.js

```diff
@@ -142,10 +157,10 @@ function initForm({ router, routeConfig }) {
     function renderForm(req, res, status = FORM_STATES.NOT_SUBMITTED) {
         const content = res.locals.content;
         const lang = req.i18n.__(routeConfig.lang);
-        const orders = get(req.session, materialsOrderKey, {});
-        const numOrders = sumBy(values(orders), order => {
-            return order.quantity;
-        });
+        const availableItems = res.locals.availableItems;
+        const orders = req.session[sessionOrderKey] || [];
+
+        const formActionBase = '/' + res.locals.sectionId + routeConfig.path;
```

Comment:
IIRC this can be written using only the request as `const formActionBase = req.baseUrl + req.path`;

Reply:
Silly follow up question, but do we need to construct this? Are we in the right context for the form to post to itself using an empty `action=''`?